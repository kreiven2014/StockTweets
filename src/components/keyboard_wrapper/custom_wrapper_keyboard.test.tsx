/* REACT */
import React from "react";

/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import CustomWrapperKeyboard from "./custom_wrapper_keyboard";
import TextInput from "src/components/base_input/base_input";

describe("CustomWrapperKeyboard tests", () => {
  test("renders correctly", (): void => {
    const tree = renderer
      .create(
        <CustomWrapperKeyboard>
          <TextInput
            placeholder="placeholder"
            value="value"
            // FUNCTIONS
            onChangeText={(value: string) => { }} // eslint-disable-line @typescript-eslint/no-unused-vars
          />
        </CustomWrapperKeyboard>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
