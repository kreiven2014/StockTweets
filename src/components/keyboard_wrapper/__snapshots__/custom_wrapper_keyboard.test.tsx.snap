// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomWrapperKeyboard tests renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "transform": Array [
        Object {
          "translateY": 0,
        },
      ],
    }
  }
>
  <View
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <TextInput
      allowFontScaling={false}
      onChangeText={[Function]}
      placeholder="placeholder"
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "borderColor": "#D7D9E0",
            "borderRadius": 4,
            "borderWidth": 1,
            "flex": 1,
            "fontSize": 24,
            "height": 68.5,
            "paddingHorizontal": 24,
          },
          undefined,
          undefined,
          Object {
            "borderColor": "#D7D9E0",
          },
        ]
      }
      underlineColorAndroid="transparent"
      value="value"
    />
  </View>
</View>
`;
